.github/workflows/main.yml

```diff
@@ -33,6 +33,8 @@ jobs:
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
See if we can get by with less than a full history fetch.

Reply:
Probably could by settings `<GitVersion_NoFetch>false</GitVersion_NoFetch>` in `build/verions/Mlos.NetCore.Version.props`, but I'm not sure it's worth it.